stdlib/public/core/String.swift

```diff
@@ -437,44 +437,44 @@ extension String {
   /// requested capacity was too small).
   ///
   /// This method replaces ill-formed UTF-8 sequences with the Unicode
-  /// replacement character (`"\u{FFFD}"`); This may require resizing
+  /// replacement character (`"\u{FFFD}"`). This may require resizing
   /// the buffer beyond its original capacity.
   ///
   /// The following examples use this initializer with the contents of two
-  /// different `UInt8` arrays---the first with well-formed UTF-8 code unit
-  /// sequences and the second with an ill-formed sequence at the end.
+  /// different `UInt8` arrays---the first with a well-formed UTF-8 code unit
+  /// sequence, and the second with an ill-formed sequence at the end.
   ///
-  ///     let validUTF8: [UInt8] = [67, 97, 102, -61, -87, 0]
-  ///     let s = String(unsafeUninitializedCapacity: validUTF8.count,
-  ///                    initializingUTF8With: { ptr in
-  ///         ptr.initializeFrom(validUTF8)
+  ///     let   validUTF8: [UInt8] = [0x43, 0x61, 0x66, 0xC3, 0xA9]
+  ///     let invalidUTF8: [UInt8] = [0x43, 0x61, 0x66, 0xC3]
+  ///
+  ///     String(unsafeUninitializedCapacity: validUTF8.count,
+  ///            initializingUTF8With: { ptr in
+  ///         _ = ptr.initialize(from: validUTF8)
   ///         return validUTF8.count
   ///     })
-  ///     // Prints "Café"
+  ///     // $R0: String = "Café"
```

Comment:
I don't think we like to use LLDB output as a convention in docs. @natecook1000 can you comment here?